content/_index.html

```diff
@@ -6,13 +6,19 @@
 
 {{< blocks/cover title="Welcome to FSH School" image_anchor="top" height="full" color="orange" >}}
 <div class="mx-auto">
+	<a class="btn btn-lg btn-secondary mr-3 mb-4" target="_blank" href="/FSHOnline/">
+		Interact with FSH <i class="fas fa-fish "></i>
```

Comment:
`Interact with FSH` is more descriptive than `Experiment`, but is there a punnier opportunity here?  `Play with FSH`?  `Visit the Touch Tank`?  (Ooh.  That would have been a good name for FSH Online too.)

Reply:
I like play.

Reply:
Changed.

Reply:
Play it is!